From 06934a1c742af772b1b9d869ae73d8948565043e Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Thu, 28 Aug 2025 17:25:38 +0200
Subject: [PATCH] patch

---
 scripts/build.py | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/scripts/build.py b/scripts/build.py
index ef555eb..826918e 100644
--- a/scripts/build.py
+++ b/scripts/build.py
@@ -34,7 +34,7 @@ def detect_arch():
             and ("libc" not in arch or arch.get("libc") == libc)
         ):
             if arch["libdir"]:
-                arch["libdir"] = os.path.expanduser(arch["libdir"])
+                arch["libdir"] = os.environ["LIBDIR_PATH"]
             else:
                 global tmpdir
                 if "CI" in os.environ:
@@ -42,8 +42,7 @@ def detect_arch():
                     os.makedirs(tmpdir, exist_ok=True)
                     arch["libdir"] = tmpdir
                 else:
-                    tmpdir = tempfile.TemporaryDirectory()
-                    arch["libdir"] = tmpdir.name
+                    arch["libdir"] = os.environ["LIBDIR_PATH"]
             return arch
     raise Exception(f"Unsupported arch: {uname}")
 
@@ -139,8 +138,6 @@ def get_curl_libraries():
 ffibuilder = FFI()
 system = platform.system()
 root_dir = Path(__file__).parent.parent
-download_libcurl()
-
 
 ffibuilder.set_source(
     "curl_cffi._wrapper",
-- 
2.39.5 (Apple Git-154)

