From 2f3cfad73747a34cf38b93ee45a0e0264a597b8a Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Wed, 23 Jul 2025 21:18:47 +0200
Subject: [PATCH] patch

---
 scripts/build.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/build.py b/scripts/build.py
index 58ec45b..af28458 100644
--- a/scripts/build.py
+++ b/scripts/build.py
@@ -42,8 +42,8 @@ def detect_arch():
                     os.makedirs(tmpdir, exist_ok=True)
                     arch["libdir"] = tmpdir
                 else:
-                    tmpdir = tempfile.TemporaryDirectory()
-                    arch["libdir"] = tmpdir.name
+                    # tmpdir = tempfile.TemporaryDirectory()
+                    arch["libdir"] = os.environ["LIBDIR_PATH"]
             return arch
     raise Exception(f"Unsupported arch: {uname}")
 
@@ -138,7 +138,7 @@ def get_curl_libraries():
 ffibuilder = FFI()
 system = platform.system()
 root_dir = Path(__file__).parent.parent
-download_libcurl()
+# download_libcurl()
 
 
 ffibuilder.set_source(
-- 
2.39.5 (Apple Git-154)

